### ISSUE TITLE
[a11y] tooltip popup should have correct semantics role

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#161630]
When using https://api.flutter.dev/flutter/material/Tooltip-class.html, the tooltip popup (not the tooltip widget) should have following role
| OS | role |
|--------|--------|
| web | tooltip |
| ios | - |
| macos | NSAccessibilityGroupRole, AXUserInterfaceTooltip |
| windows | ROLE_SYSTEM_TOOLTIP |
| android | - |

**Comment by victorsanni** [comment]
Now that RawTooltip has been added, RawTooltip.tooltipBuilder should have SemanticsRole.tooltip.

**Comment by elliette** [comment]
Hi @chunhtai @victorsanni  - Some of the integration tests in DevTools have begun failing with the following error ([example issue](https://github.com/flutter/devtools/issues/9639)):

```
The following assertion was thrown during a scheduler callback:
Invisible SemanticsNodes should not be added to the tree.
The following invisible SemanticsNodes were added to the tree:
SemanticsNode#107(Rect.fromLTRB(1461.0, 5.0, 1489.0, 25.0), flags: [isButton, hasEnabledState, isEnabled, isFocusable])
which was added as a child of:
  SemanticsNode#108(Rect.fromLTRB(1401.0, 5.0, 1489.0, 25.0), tags: [_InputDecoratorState.suffixIcon], traversalParentIdentifier: _OverlayPortalState#6269d(lifecycle state: defunct, not mounted), tooltip: "Help", textDirection: ltr)
An invisible SemanticsNode is one whose rect is not on screen
hence not reachable for users, and its semantic information is
not merged into a visible parent.
An invisible SemanticsNode makes the accessibility experience
confusing, as it does not provide any visual indication when the
user selects it via accessibility technologies.
Consider removing the above invisible SemanticsNodes if they were
added by your RenderObject.assembleSemanticsNode implementation,
or filing a bug on GitHub:
  https://github.com/flutter/flutter/issues/new?template=02_bug.yml
```

I've been able to track down the widget its complaining about to:

==> [`InputDecorationSuffixButton.help`](https://github.com/flutter/devtools/blob/a525258ed3b827d967970c8c68005ae7596f55c0/packages/devtools_app_shared/lib/src/ui/text_field.dart#L126-L133)
 ====> which is [wrapped in a tooltip](https://github.com/flutter/devtools/blob/a525258ed3b827d967970c8c68005ae7596f55c0/packages/devtools_app_shared/lib/src/ui/text_field.dart#L141) via our helper [`maybeWrapWithTooltip` function](https://github.com/flutter/devtools/blob/a525258ed3b827d967970c8c68005ae7596f55c0/packages/devtools_app_shared/lib/src/ui/common.dart#L485-L498)
======> Which uses Flutter's [`Tooltip`](https://github.com/flutter/devtools/blob/a525258ed3b827d967970c8c68005ae7596f55c0/packages/devtools_app_shared/lib/src/ui/common.dart#L383-L393) (see screenshot below)

<img width="508" height="92" alt="Image" src="https://github.com/user-attachments/assets/3a9ee7c0-89fd-48a9-8abc-b229a7ab1642" />

Could the error we are hitting be related to this issue? (And if so, is there a workaround we could try?) Thank you!

**Comment by elliette** [comment]
Hi @victorsanni - yes, you can run https://github.com/flutter/devtools/blob/a525258ed3b827d967970c8c68005ae7596f55c0/packages/devtools_app/integration_test/test/live_connection/app_test.dart#L12 (you will need to install ChromeDriver if you don't already have it installed)

**Comment by elliette** [comment]
Oh and I forgot - you will also need to remove it from this disabled tests list, otherwise running the above command will skip the entire test: https://github.com/flutter/devtools/blob/a525258ed3b827d967970c8c68005ae7596f55c0/packages/devtools_app/integration_test/run_tests.dart#L44

Thank you!
